modules/network/src/main/java/org/apache/ignite/internal/network/DefaultMessagingService.java

```diff
@@ -418,17 +418,29 @@ private void handleMessageFromNetwork(InNetworkObject inNetworkObject) {
                 logAndRethrowIfError(inNetworkObject, e);
             } finally {
                 long tookMillis = TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - startedNanos);
+
                 if (tookMillis > 100) {
-                    LOG.warn("Processing of {} from {} took {} ms", inNetworkObject.message(), inNetworkObject.sender(), tookMillis);
+                    LOG.warn(
+                            "Processing of {} from {} took {} ms",
+                            LOG.isDebugEnabled() ? inNetworkObject.message() : inNetworkObject.message().toStringForLightLogging(),
```

Comment:
Should we also add `&& includeSensitive()` here?

Reply:
Fix it